update docs for team admin permissions++ (#5564)
pulumi/docs: content/docs/intro/console/collaboration/teams.md

Teams allow organization admins to assign a set of stack permissions
to a group of users.

## Team Roles

> Members of a team can be granted `Team admin` or `Team member` permissions. Team admin's can add members to a

Reviewer: ```suggestion
Members of a team can be granted `Team admin` or `Team member` permissions. Team admins can add members to a
```

> Members of a team, can be granted `Team admin` or `Team member` permissions. Team admin's can add members to a
> team. Both team admins and team members can grant stack access to a team. By default any new team members will be
> assinged the team member role. To change a team member's role use the 3-dot menu at the end of the table row.

Author: ```suggestion
Members of a team can be granted `Team admin` or `Team member` permissions. Team admin's can add members to a
team. Both team admins and team members can grant stack access to a team. By default, any new team members will be
assigned the team member role. To change a team member's role use the 3-dot menu at the end of the table row.
```

> assigned the team member role. To change a team member's role use the 3-dot menu at the end of the table row.

Reviewer 2: Does three-dot mean ellipsis in this context?

```suggestion
assigned the team member role. To change a team member's role use the ellipsis menu item at the end of the table row.
```

Author: yes

> assigned the team member role. To change a team member's role use the ellipsis menu item at the end of the table row.

Reviewer: ```suggestion
assigned the team member role. To change a team member's role, use the ellipsis menu item at the end of the table row.
```

### Creating a Team

> Organization admins can add a new team by going to the organization's **Teams** tab, and then
> selecting **Create team**. Organization members can also be granted permissions to create teams

Reviewer 2: ```suggestion
Organization admins can add a new team by going to the organization's **Teams** tab and 
selecting **Create team**. Organization members can also be granted permissions to create teams
```

> Organization admins can add a new team by going to the organization's **Teams** tab, and then
> selecting **Create team**. Organization members can also be granted permissions to create teams
> in the organziation settings. Any member who creates a team will automatically be assinged the team admin role.

Author: ```suggestion
Organization admins can add a new team by going to the organization's **Teams** tab, and then
selecting **Create team**. Organization members can also be granted permissions to create teams
in the organization settings. Any member who creates a team will automatically be assigned the team admin role.
```

> in the organization settings. Any member who creates a team will automatically be assigned the team admin role.

Reviewer: Should this maybe be

```suggestion
in the organization's Settings tab. Any member who creates a team will automatically be assigned the team admin role.
```

? It just reads a bit oddly as "in the organization settings."
